src/Core/Utility/Path.re

```diff
@@ -38,3 +38,15 @@ let trimTrailingSeparator = path => {
     path;
   };
 };
+
+let getExtension = path => {
+  let fileName = Rench.Path.filename(path);
+
+  if (String.length(fileName) == 0) {
+    "";
+  } else if (fileName.[0] == '.') {
+    fileName;
+  } else {
+    Rench.Path.extname(fileName);
+  };
+};
```

Comment:
So I was wrong! The reason that the `.bashrc` and a few other things weren't working was because of a conflict between `Rench.Path.extname` and what VSCode considers an extension.

VSCode considers `.bashrc` a file extension (to an unnamed file I guess, I don't follow the logic and assumed that `.bashrc` would have been a `filename` 🤷 Do people have files with it as an extension? ), whereas Rench would return an empty string for `.bashrc`.

Now I just return the full filename if the first character is a `.`, which fixes the issue but not sure if there is some edge case I'm missing.

Reply:
> VSCode considers .bashrc a file extension (to an unnamed file I guess, I don't follow the logic and assumed that .bashrc would have been a filename 🤷 Do people have files with it as an extension? ), whereas Rench would return an empty string for .bashrc.

Oh, nice catch @CrossR ! That would indeed be a problem. It does seem counter-intuitive (I wouldn't expect '.bashrc' to be considered an extension either) - but being consistent with how VSCode evaluates would be the correct behavior here. This logic looks reasonable to me